### ISSUE TITLE
LOSS go to NAN in Batch1 sample2.

### ISSUE THREAD

**Issue by Chizuru-Kagura** [alexmagsam/keras-rpn#2]
Hi, sorry to disturb you but it not working on my sever.  
When I ran it,  told me in 
Batch1/10(1/83),  loss: 11.3374 - rpn_class_loss: 6.3124 - rpn_bbox_loss: 5.0250 (random sample). 
But in Batch1/10(2/83),  loss, run_class_loss, rpn_bbox_loss all go to NAN. I tried to decrease learning rate but it not work. I didn't change anying in your project. What should I do next or what point I'm not attention?
**==================================================================================================
Total params: 28,121,490
Trainable params: 28,068,370
Non-trainable params: 53,120
__________________________________________________________________________________________________
None
Epoch 1/10
4/83 [>.............................] - ETA: 2:10:47 - loss: nan - rpn_class_loss: nan - rpn_bbox_loss: nan**

PS: What's "None" means in foregoing print? Is that will has some influence  with all loss to be NAN?
Again thanks to your sharing.

**Comment by Puranjay-del-Mishra** [comment]
Did you solve this problem?
